packages/lightning-lsp-common/src/resources/sfdx/typings/lds.d.ts

```diff
@@ -573,6 +609,20 @@ declare module 'lightning/analyticsWaveApi' {
     // eslint-disable-next-line @typescript-eslint/no-empty-interface
     export interface AssetReferenceRepresentation extends BaseAssetReferenceRepresentation {}
 
+    /**
+     * Base Wave Asset input Representation
```

Comment:
Change Wave to Tableau CRM

Reply:
it seems there are many places using Wave now in this file. Do we want to change them all?